build_tools/cmake/iree_bytecode_module.cmake

```diff
@@ -96,11 +96,30 @@ function(iree_bytecode_module)
     get_filename_component(_FRIENDLY_NAME "${_RULE_SRC}" NAME)
   endif()
 
+  # Check LLVM static library setting. If the static libary output path is set,
+  # retrieve the object path and the corresponding header file path.
+  foreach(_FLAG ${_RULE_FLAGS})
```

Comment:
Rather than introspecting this out of the flags, could we add it as an option to the CMake function? That seems cleaner

Reply:
Since we need to specify the static library path in the `iree-compiler` flags anyway, traversing through the compiler flags seems to be cleaner, otherwise the new option will have the same value as the `iree-llvm-static-library-output-path`.

Reply:
Yeah, I mean it would be used to add `iree-llvm-static-library-output-path` to the flags. In the same way that we have `IREE_BYTECODE_MODULE_ENABLE_TSAN` add the tsan flag, for instance

Reply:
ahh... got it. I will send out a PR to add the feature. :)